Contributing.md

```diff
@@ -0,0 +1,92 @@
+# Contribution Guide
+
+If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🥳
+
+# How to Contribute
+
+1. Fork this repository.
+
+2. Clone the forked repository.
+
+   ```bash
+   git clone 'https://github.com/bsoc-bitbyte/busify.git'
+   ```
+
+3. Navigate to the project directory.
+
+   ```bash
+   cd busify
+   ```
+
+4. Make changes in source code.
+
+5. Stage your changes and commit
+
+6. git add .
+
+7. git commit -m "<your_commit_message>"
+
+8. Push your local commits to the remote repo.
+
+9. git push
+
+10. Create a PR to develop repository.
+
+## Conventional Commit Messages
+
+In our project, we follow the convention of using conventional commit messages for all commits. Conventional commit messages provide a standardized format for commit messages, making it easier to understand and track changes in our codebase.
+
+A conventional commit message consists of a concise and structured format:
+
+<type>: <description>
```

Comment:
The format for commit message is 
`<type>(optional_scope): <your_commit_message>`